jina/flow/base.py

```diff
@@ -31,7 +32,8 @@
 
 __all__ = ['BaseFlow']
 
-from ..peapods import BasePod
+from ..peapods import Pod, BasePod
```

Comment:
`BasePod` should go to a `if False:` block, since it is only used for type checking.

Reply:
done